tests/App/Entity/Gallery.php

```diff
@@ -23,16 +23,15 @@
 class Gallery extends BaseGallery
 {
     /**
+     * @var int|null
+     *
```

Comment:
```suggestion
```

Reply:
We can't, on master that typehint is removed because the getId() is a method that comes from the interface and it can accept both int/string.